test/baselines/sources/structured-headers.html

```diff
@@ -23,6 +23,8 @@ <h1>
   </emu-alg>
 </emu-clause>
 
+<!-- FIXME: Shouldn't this warning be suppressed by `redefinition`? -->
```

Comment:
No - `id`s have effects on the document considered as a piece of HTML (i.e., for computers), not just on the document considered as a piece of writing (i.e., for humans).

"redefinition" means that we are redefining an algorithm, which is normally an error when considering the document as a piece of writing (since a human reader expects a given term to have only one definition). But it doesn't have any interaction with the error of reusing an `id`, since that's a problem for the _computer_, not just for humans: the computer needs distinct `id`s for each clause so that you can link to them independently.